app/schema.graphql

```diff
@@ -0,0 +1,103 @@
+scalar Datetime
+
+type Author {
+  name: String
+  email: String
+}
+
+# git commit data.
+type Commit {
+  sha1: String!
```

Comment:
This can be calculated via [`Commit.id`](https://docs.rs/git2/0.10.1/git2/struct.Commit.html#method.id) to get an `Oid`. Then [`Oid.as_bytes`](https://docs.rs/git2/0.10.1/git2/struct.Oid.html#method.as_bytes) to get the byte slice.